client/devdocs/docs-example/readme-viewer.jsx

```diff
@@ -19,21 +19,25 @@ class ReadmeViewer extends Component {
 	static propTypes = {
 		getReadme: PropTypes.func,
 		readmeFilePath: PropTypes.string,
+		section: PropTypes.string,
 	};
 
 	static defaultProps = {
-		getReadme: readmeFilePath => {
+		getReadme: ( section, readmeFilePath ) => {
```

Comment:
This might be a bit cleaner as:

```js
getReadme: ( section, readmeFilePath ) => {
	const sectionPath = section === 'blocks' ? 'blocks' : 'components';
	return htmlToReactParser.parse( require( `../../${ sectionPath }/${ readmeFilePath }/README.md` ) );
}
```

Reply:
Webpack is forced to put more into the chunk the more ambiguous the dynamic require is. See [the docs](https://webpack.js.org/guides/dependency-management/#require-with-expression) on require statements that contain expressions within them.

Essentially `../../blocks` is much better than `../../${ variableThatMightEvalToAnything }`

Reply:
Ah right, that makes sense. I haven't been doing much with dynamic require and webpack so I didn't even consider this–cheers! 😄